dev-support/create-release/release-util.sh

```diff
@@ -678,8 +678,8 @@ function kick_gpg_agent {
 # Do maven command to set version into local pom
 function maven_set_version { #input: <version_to_set>
   local this_version="$1"
-  log "${MVN[@]}" versions:set -DnewVersion="$this_version"
-  "${MVN[@]}" versions:set -DnewVersion="$this_version" | grep -v "no value" # silence logs
+  log "${MVN[@]}" versions:set-property -Dproperty=revision -DnewVersion="$this_version" -DgenerateBackupPoms=false
```

Comment:
The problem of using verisons plugin is that, it always requires updating the version. But here we only want to change the revision property, leave the version with ${revision} placeholder...

Reply:
In this patch, I'm using `versions:set-property`, not `versions:set`. `set-property` does the appropriate thing, try it.

Reply:
Ah, tried locally, seems OK, the -DnewVersion=xxx made me think it is going to set the version field, but actually, it is used to specify the value of the property...